### ISSUE TITLE
Request for a quick installation and setup HowTo for lnx_backup

### ISSUE THREAD

**Issue by themelle** [mape2k/check_mk_extensions#6]
Dear Marcel,

I'm glad I found your lnx_backup check plugin, ad I am grateful for the work you've already done.

We are using check-mk raw edition, and I was able to install lnx_backup-2.0.mkp through the command line.

Also, I was able to figure out the basic usage, e.g.
`/opt/omd/sites/MYSITE/local/share/check_mk/agents/lnx_backup BACKUP_NAME duply duply default status`
gathers backup status and statistics and writes this data to 
`/var/lib/check_mk_agent/lnx_backup/BACKUP_NAME`
, while 
`/opt/omd/sites/MYSITE/local/share/check_mk/agents/plugins/lnx_backup`
is the actual agent plug-in which inserts the data from 
`/var/lib/check_mk_agent/lnx_backup/BACKUP_NAME`
into the check-mk-agent output.

So, I assume I have to first download
`/opt/omd/sites/MYSITE/local/share/check_mk/agents/plugins/lnx_backup`
to the monitored host and place it into
`/usr/lib/check_mk_agent/plugins/`.

But what should I do with 
`/opt/omd/sites/MYSITE/local/share/check_mk/agents/lnx_backup`
? 

Should I use it as a wrapper for the actual duply cronjob on the monitored machine?
Or should it run in parallel? 
Or is it invoked by check_mk_agent by any means? 

That''s the part that is not really clear to me.

Thanks for your help, 
and all the best for 2023!
Andreas

**Comment by mape2k** [comment]
Thanks for your request.
You're right. You have to use agents/lnx_backup as a wrapper for rsync, tar and so on...

e.g. if your using rsync for your backup
lnx_backup My_Backup_Name rsync rsync --stats --other-args ...

where...
My_Backup_Name is the Name of the backup - every backup results in a single check item on check_mk
(first) rsync is the type of backup mechanism, so the wrapper could parse output correctly
(second) rsync and following arguments are your original command line for the backup

Hope this helps. I'll leave the issue to add some quick installation guide in future.

**Comment by mape2k** [comment]
Fixed in version 2.4.0
